RxCocoa/Common/DelegateProxyType.swift

```diff
@@ -360,10 +360,11 @@ extension DelegateProxyType where ParentObject: HasPrefetchDataSource, Self.Dele
                     
                 return Disposables.create { [weak object] in
                     subscription.dispose()
-
-                    if object?.window != nil {
-                        object?.layoutIfNeeded()
-                    }
+                    MainScheduler.assumeMainActor(execute: {
```

Comment:
The behavior is not changed. `assumeMainActor` just silences the compiler warning. This should be executed on `MainScheduler` because `object?.window` and `object?.layoutIfNeeded()` can only be called from the main thread.

I'll submit a fix to ensure that code is run on the main thread.